builtin/game/mod_profiling.lua

```diff
@@ -67,16 +64,17 @@ function mod_statistics.update_statistics(dtime)
 				max_per = 0,
 				avg_per = 0
 			}
+			mod_statistics.stats[name] = stats
 		end
 		
-		local modtime = 0
-		for type,time in pairs(types) do
-			modtime = modtime + time
-			if mod_statistics.stats[modname].types == nil then
-				mod_statistics.stats[modname].types = {}
```

Comment:
modtime is perfectly fine naming style if we're using something similar to the Linux kernel guidelines